packages/ods/src/components/tabs/src/components/ods-tab/ods-tab.scss

```diff
@@ -3,6 +3,7 @@
 
 :host(.ods-tab) {
   display: block;
+  margin-bottom: -2px;
```

Comment:
please use a shared variable for `margin-bottom` and `border-bottom` size here and `border-bottom` size in ods-tabs.scss